metagpt/utils/common.py

```diff
@@ -49,7 +49,7 @@ def parse_blocks(cls, text: str):
 
     @classmethod
     def parse_code(cls, text: str, lang: str = "") -> str:
-        pattern = rf'```{lang}.*?\s+(.*?)```'
+        pattern = rf'```{lang}.*?\s+(.*)```'
```

Comment:
make sure it won't influence existing code parsing strategy

Reply:
This function `OutputParser.parse_code` is only refered here.